pkg/virt-handler/vm.go

```diff
@@ -1271,10 +1271,21 @@ func (d *VirtualMachineController) checkNetworkInterfacesForMigration(vmi *v1.Vi
 	for _, network := range vmi.Spec.Networks {
 		networks[network.Name] = network.DeepCopy()
 	}
+	// if no networks or interfaces are found, return error
+	// this can be in the case of a new minimal VMI
+	if len(networks) == 0 || len(vmi.Spec.Domain.Devices.Interfaces) == 0 {
+		return fmt.Errorf("cannot migrate VMI, network or interfaces not found")
```

Comment:
That is not the case. There is the implicitly set configuration with interface connected to the pod network via bridge binding. Please mention in the comment that the default binding (when nothing else is specified) is bridge.

Also, there is also a flag that users can use to really have *no* interfaces at all. Then we should not block the migration. I don't recall what the flag is named though. Maybe @rmohr can help here?